api/urls.py

```diff
@@ -3,10 +3,12 @@
 
 from api.views.song import SongViewSet
 from api.views.song_entry import SongEntryViewSet
+from api.views.songbook import SongbookViewSet
 
 router = routers.DefaultRouter()
 
 router.register(r"songs", SongViewSet)
-router.register(r"song_entry", SongEntryViewSet)
+router.register(r"song_entries", SongEntryViewSet)
```

Comment:
And I asked him to pluralize this for consistency (and generally, plurals are the accepted way to name endpoints)